src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const sum = days * 40;
```

Comment:
You use "magic numbers" in your code. I think it's better to assign all constant values to variables, and use variables in formulas. But the checklist says nothing about it. Therefore, approve.